values-changes.yaml

```diff
@@ -82,3 +82,11 @@ changes:
     additions:
       - databases.keycloak.useOtomiDB: false
       - databases.keycloak.imported: false
+  - version: 13
+    additions:
```

Comment:
The `additions` are not necessary, because changes defined here are about the values-schema changes not chart directories added

Reply:
Updated values-changes.yaml